tests/ociswrapper/wrapper/handlers/handler.go

```diff
@@ -113,7 +113,8 @@ func RollbackHandler(res http.ResponseWriter, req *http.Request) {
 
 	var message string
 	ocis.EnvConfigs = []string{}
-	success, _ := ocis.Restart(os.Environ())
+
+	success, _ := ocis.Restart([]string{})
```

Comment:
got it now, then maybe you can simply use it this because the assigned is the same. see the line just above it
```suggestion
	success, _ := ocis.Restart(ocis.EnvConfigs)
```